src/Makefile.test.include

```diff
@@ -104,6 +104,7 @@ UNITE_TESTS =\
   test/multisig_tests.cpp \
   test/net_tests.cpp \
   test/netbase_tests.cpp \
+  test/new_pos_valid_block_tests.cpp \
```

Comment:
I am surprised if this works, to my understanding this should break the build in case `--disable-wallet` as it uses the `TestChain100Setup` which AFAIK is only available in a wallet build and should therefor go to the wallet section of tests.

Reply:
Thanks, completely forgot about this(

Reply:
It failed indeed: https://travis-ci.com/dtr-org/unit-e/jobs/203293768